### ISSUE TITLE
Shipped plugin not found regression

### ISSUE THREAD

**Issue by marksvc** [audacity/audacity#1906]
**Describe the bug**

Audacity 3.0.2 shows mod-script-pipe at Edit - Preferences, Modules.
Audacity 3.0.3 thru 3.0.5 says "No modules were found".

I observe that Audacity 3.0.2 has file .../share/audacity/modules/mod-script-pipe.so, while Audacity 3.0.3+ has file .../lib/audacity/modules/mod-script-pipe.so. (This might have been from https://github.com/audacity/audacity/pull/919 ?)

If I do the following, it makes the mod-script-pipe module show up in Edit - Preferences, Modules, for Audacity 3.0.5:
```bash
mkdir -p .../share/audacity/modules/mod-script-pipe.so
mv .../lib/audacity/modules/mod-script-pipe.so .../share/audacity/modules/mod-script-pipe.so
```

**To Reproduce**
1. Click Edit, Preferences, Modules.
2. See message "No modules were found".

**Expected behavior**
mod-script-pipe is present and I can enable it.

**Screenshots**

Audacity 3.0.5 shows "No modules were found":

![audacity-no-modules-were-found](https://user-images.githubusercontent.com/7265309/137039154-c1df0366-3153-4d99-9a68-f589158c5fe5.png)

Moving the file from .../lib/... to .../share/... results in the module re-appearing:

![audacity-mod-script-pipe](https://user-images.githubusercontent.com/7265309/137039216-86a3ea0e-9196-44cf-a84e-86574166745d.png)

**Additional information (please complete the following information):**
 - OS: Ubuntu 20.04, using Audacity installed from flatpak
 - Version 3.0.5

**Comment by marksvc** [comment]
I still see the same problem with the newer version of Audacity. I downloaded the Audacity 3.1.0 AppImage. In Preferences: Modules, I see "No modules were found".

This is using Ubuntu 20.04. 

![audacity-3 1 0-modules](https://user-images.githubusercontent.com/7265309/140790760-d9567bbc-d1c9-4d8e-b677-faaa1ea1aef4.png)

**Comment by LWinterberg** [comment]
this got fixed in #2522
